pkgs/applications/misc/bdt/default.nix

```
@@ -0,0 +1,21 @@
+{ pkgs ? import <nixpkgs> { }, lib ? pkgs.lib }:
+pkgs.rustPlatform.buildRustPackage rec {
+  pname = "bdt";
+  version = "0.18.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "datafusion-contrib";
+    repo = pname;
```

```suggestion
    repo = "bdt";
```

reusing pname here is discouraged